CHANGELOG.md

```diff
@@ -13,17 +13,18 @@ However, some entries may be manually edited, where it helps for clarity and und
 ## v0.6.0 (2022-05-27)
 ### Feature
 * Provide an option to force analysis ([#55](https://github.com/phylum-dev/phylum-ci/pull/55)) ([`4d6fc3b`](https://github.com/phylum-dev/phylum-ci/commit/4d6fc3b842cec004d655d1c1a63553a0c54e1d54))
+* Default to project settings for risk domain thresholds ([#52](https://github.com/phylum-dev/phylum-ci/pull/52)) ([`9f10442`](https://github.com/phylum-dev/phylum-ci/commit/9f10442ba41300093c65a5e5e1ff2fdb71c0772e))
+* Default to analyzing new dependencies only ([#53](https://github.com/phylum-dev/phylum-ci/pull/53)) ([`e0894fc`](https://github.com/phylum-dev/phylum-ci/commit/e0894fcf9f52d3014798f8676a5ff2360e725a8a))
 
 ### Fix
 * Ensure the "CI Platform Name" portion of a label is correct ([#55](https://github.com/phylum-dev/phylum-ci/pull/55)) ([`1867fb6`](https://github.com/phylum-dev/phylum-ci/commit/1867fb6e543183aa894cec4e06828069d62dee01))
 * Enable Phylum UI links for groups ([#54](https://github.com/phylum-dev/phylum-ci/issues/54)) ([`8775a63`](https://github.com/phylum-dev/phylum-ci/commit/8775a6392456fe64f97efae7f8d514ebf66f6949))
 
 ### Breaking Changes
-* Default to project settings for risk domain thresholds ([#52](https://github.com/phylum-dev/phylum-ci/pull/52)) ([`9f10442`](https://github.com/phylum-dev/phylum-ci/commit/9f10442ba41300093c65a5e5e1ff2fdb71c0772e))
-  * Individual risk domain threshold values can be set with command line options, which now accept values between 0 and 100, inclusive
+* Individual risk domain threshold values can be set with command line options, which now accept values between 0 and 100, inclusive
   * Previously, the accepted values were between 0 and 99, inclusive
-* Default to analyzing new dependencies only ([#53](https://github.com/phylum-dev/phylum-ci/pull/53)) ([`e0894fc`](https://github.com/phylum-dev/phylum-ci/commit/e0894fcf9f52d3014798f8676a5ff2360e725a8a))
-  * The option to analyze `--new-deps-only` was removed and replaced with one that has the opposite meaning: `--all-deps`
+* The option to analyze `--new-deps-only` was removed and replaced with one that has the opposite meaning: `--all-deps`
+* The short option to `--force-install` was changed from `-f` to `-fi`
```

Comment:
Do the other options that change need to be documented too?

Reply:
The change here was for the previous release, v0.6.0. The changes in this PR will be included in the v0.7.0 changelog entry...which gets populated automatically during release. The reason for the manual change here was due to the way `python-semantic-release` missed the BREAKING CHANGES for the last release...which I'm suspected now was my fault since I used `!feat` instead of `feat!` to represent the commits with BREAKING CHANGES. The intent was just to clean things up a bit now.